### PR TITLE
Artifact menu arrangement

### PR DESCRIPTION
**Pull request by markkramerus**
Major upgrade to the organization of the IG.

### CODE REVIEW

**Comment by markkramerus** [comment]
I would like at least 1 review for this.

File: input/fsh/VS_ComorbidCondition.fsh [hunk 1]
```diff
@@ -1,5 +1,5 @@
 CodeSystem: ElixhauserAHRQ_CS
-Id: mcode-elixhauser-category-cs
+Id: elixhauser-ahrq-cs
```
Comment (mlterryMitre) [inline input/fsh/VS_ComorbidCondition.fsh:2]: minor comment, specifying ahrq might be unnecessary if Elixhauser is maintained standards-wise only by AHRQ.

File: input/pagecontent/examples.md [hunk 2]
```diff
@@ -78,7 +44,7 @@ The patient is two years out from surgery and has undergone surveillance imaging
 * [`teleradiotherapy-prescription-delivery-jenny-m`](Procedure-teleradiotherapy-prescription-delivery-jenny-m.html) ([TeleradiotherapyPrescriptionDelivery])
```
Comment (mlterryMitre) [inline input/pagecontent/examples.md:44]: the the “([])” presentation renders with these characters in the Extended Example. Is this on purpose or an error in markdown formatting?

**Review by mlterryMitre — CHANGES_REQUESTED** [review]
nice re-organization of menu + content by group option.
I just have some minor cosmetic comments in-line.

File: input/fsh/VS_ComorbidCondition.fsh (hunk 1 above)
Comment (markkramerus) [reply input/fsh/VS_ComorbidCondition.fsh:2]: It turns out there are more than one Elixhauser. There's an AHRQ version, a Manitoba version, and more. So it seems necessary to specify.

File: input/pagecontent/examples.md (hunk 2 above)
Comment (markkramerus) [reply input/pagecontent/examples.md:44]: Left out the "include" needed to pull in the references. Thanks for catching this.

**Pull request merged**